docker-compose.yml

```diff
@@ -1,34 +1,33 @@
 version: "3.8"
 services:
-    web:
-        build: .
-        ports:
-            - "8000:8000"
-        command: npx nodemon -L app.js
-        volumes:
-            - ./app:/usr/src/app
-            - /usr/src/app/node_modules
-            - ./.git/FETCH_HEAD:/usr/src/app/FETCH_HEAD
-        env_file: 
-            - .env
-        depends_on:
-            - mongo
-        
-    mongo:
-        image: mongo:latest
-        container_name: mongo_db
-        volumes:
-            - ./config/init.sh:/docker-entrypoint-initdb.d/init.sh
-            - ./config/mongod.conf:/etc/mongod.conf
-            - ./logs:/var/log/mongodb/
-            - /data/db
-        env_file:
-            - .env
-        ports:
-            - "27017:27017"
-        restart: on-failure:5
-        command: ["mongod", "-f", "/etc/mongod.conf"]
+  web:
+    build: .
+    ports:
+      - "8000:8000"
+    command: npx nodemon -L app/app.js
+    volumes:
+      - ./app:/usr/articleblog/app
+    environment:
+      - COMMIT_HASH
```

Comment:
It might crash then if someone will simply run `docker-compose up` instead of `npm run up`, can you add at the top of readme that one is supposed to run `npm run up`?

Reply:
Ah actually, we've had a conversation about it with Jordan earlier on, He also wanted to include something in the npm scripts. The problem is that once you do it, the environment where we deploy our app has to have NodeJS installed, since it's outside the container, and we decided that it's a bit uncool to add dependency needed outside the container just to add commit hash, so it might be better at the end of the day to stick with mounting the `.git`

Reply:
Of which solution I'm also not a fan of, so if you'd find a way to include this env variable in a way that would not require the host machine to install new dependencies, I'd be more than happy.

Reply:
It's a fair point. The reason I'm fetching Git commit hash outside docker-compose is that docker-compose doesn't seem to be able to execute shell commands. I found no way to put `$(git rev-list -1 HEAD)` directly in `docker-compose.yml`.

A dependency-free way of doing similar thing would be to make an `up.sh` script with contents just the same as the contents of the "up" NPM script:
```bash
#!/bin/sh
COMMIT_HASH=$(git rev-list -1 HEAD) docker-compose up --build
```

Reply:
Wouldn't it cause additional issues on Windows @bilgorajskim?

Reply:
Yes. I haven't thought of that. You can't run .sh script on native Windows terminal. But you absolutely can on WSL1/WSL2. Perhaps it is good enough to require the user to use WSL. I think Docker for Windows is deprecating the non-WSL version anyway (the Hyper-V backend is marked as legacy in Docker app), so soon to use Docker on Windows, the user would have to install WSL anyway.

![Docker_Desktop_rvnqixNBhK](https://user-images.githubusercontent.com/5052636/104918895-bb29a580-5995-11eb-9599-7612fd0f164f.png)

Reply:
Alright could you use the sh solution? I approve the PR in advance. Please merge once done

Reply:
Sure, thanks.